downstream/modules/hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -8,10 +8,9 @@ After you sign and approve collections, download the signature public keys from
 You must download the public key before you add it to the local system keyring.
 
 .Procedure
-
-. Log in to your {PlatformNameShort}.
-include::snippets/snip_menu-top-ac.adoc[]
-*{MenuACAdminSignatureKeys}*.
+//[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
+. Log in to your {HubName}.
+. From the navigation panel, select {MenuACAdminSignatureKeys}.
```

Comment:
this one is showing as menu:Signature Keys.

Reply:
Fixed.